friktion/main.py

```diff
@@ -111,13 +115,17 @@ async def main_def():
         bid_size=1,
         order_id=order_id,
         signer_wallet=COUNTERPARTY,
+        referrer=REFERRER
     )
+
+    # happens outside of paradigm
+    (msg, signature) = bid_details.as_signed_msg(wallet)
+
     # fill offer via bid
-    if error := await c.validate_bid(swap_order_creator, bid_details):
+    if error := await c.validate_bid(swap_order_creator, bid_details, msg, str(signature)):
```

Comment:
`str(signature)` is wrapping the bytes for the message and the bytes for signature itself into a string. We won't be able to do the same in our workflow as we only expect a hexadecimal string from our users on `/quote` endpoint.

![Screen Shot 2022-07-28 at 10 29 09](https://user-images.githubusercontent.com/4041238/181517156-c5ddd886-d7c5-4497-ac26-287503a73f9f.png)

![Screen Shot 2022-07-27 at 10 07 58](https://user-images.githubusercontent.com/4041238/181517635-ec28bd8c-9be0-4d91-9d40-f243414919fe.png)

Reply:
getting this error when trying to run the script
![Screen Shot 2022-07-28 at 10 44 48](https://user-images.githubusercontent.com/4041238/181520643-547276df-4d96-498b-a928-7da252c7caf3.png)

Reply:
can you try one more time for me? I can't recreate, might have been my dependency management when testing in initial PR